### PR TITLE
Added Infinite scroll 

### LINKED ISSUE #86: [FEATURE] add infinite scroll to the projects page [FE]
### Description

the idea is to not use pagination where the UX would be not that great. Infinite Scroll is the new trend so let's implement this. The data should be getting from backend and on each `scroll down to bottom` it will fetch some extra data to the projects array. Let's discuss it if you have any doubts. 

**Note**
> This is only for frontend, backend implementation will be in different `github issue`

### PR DESCRIPTION
**Pull request by leewan-09**
### Fixes Issue #86 
closes #86 

### Changes proposed

- Added Infinite scroll

### CODE REVIEW

**Review by rohitdasu — CHANGES_REQUESTED** [review]

File: components/Projects/ProjectsList.tsx [hunk 1]
```diff
@@ -28,17 +62,24 @@ export const ProjectsList = () => {
...
-          {data.results?.map((project: IProject) => (
+          {paginatedPosts?.map((project: IProject, i: number) => (
             <Project
-              key={project.id}
+              key={i}
               description={project.description}
               title={project.title}
               tags={project.tags}
-              author={project.author.name}
+              author="TEST"
```
[the remark this answers is not in the document]
Comment (leewan-09) [reply components/Projects/ProjectsList.tsx:71]: I forgot to change, good catch

**Comment by rohitdasu** [comment]
could you check why this error is coming when i scroll down @leewan-09 

Screenshot 2022-10-01 at 3 00 49 PM

**Comment by leewan-09** [comment]
> could you check why this error is coming when i scroll down @leewan-09
> 
> Screenshot 2022-10-01 at 3 00 49 PM

problem was caused by API. author name was undefined

**Review by rohitdasu — APPROVED** [review]
LGTM 👍 
Great PR 🎉

**Pull request merged**